src/libraries/Common/src/Interop/Browser/Interop.Runtime.cs

```diff
@@ -12,31 +12,37 @@ internal static partial class Interop
     internal static unsafe partial class Runtime
     {
         [MethodImplAttribute(MethodImplOptions.InternalCall)]
-        internal static extern void ReleaseCSOwnedObject(IntPtr jsHandle);
+        internal static extern void ReleaseCSOwnedObject(nint jsHandle);
         [MethodImpl(MethodImplOptions.InternalCall)]
-        public static extern unsafe void BindJSImport(void* signature, out int is_exception, out object result);
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        public static extern void InvokeJSFunction(int functionHandle, void* data);
-        [MethodImpl(MethodImplOptions.InternalCall)]
-        public static extern void InvokeJSImport(int importHandle, void* data);
-
+        public static extern void InvokeJSFunction(nint functionHandle, nint data);
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern unsafe void BindCSFunction(in string fully_qualified_name, int signature_hash, void* signature, out int is_exception, out object result);
         [MethodImpl(MethodImplOptions.InternalCall)]
-        public static extern void ResolveOrRejectPromise(void* data);
+        public static extern void ResolveOrRejectPromise(nint data);
 
 #if !ENABLE_JS_INTEROP_BY_VALUE
         [MethodImpl(MethodImplOptions.InternalCall)]
-        public static extern IntPtr RegisterGCRoot(IntPtr start, int bytesSize, IntPtr name);
+        public static extern nint RegisterGCRoot(nint start, int bytesSize, IntPtr name);
```

Comment:
I'm not sure I like switching from IntPtr to nint for things that are pointers

Reply:
this is a alias for the same type, do you prefer `IntPtr` syntax ?